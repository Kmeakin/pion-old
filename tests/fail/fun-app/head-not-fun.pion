//~ run = "${PION} elab ${FILE}"

Bool false
