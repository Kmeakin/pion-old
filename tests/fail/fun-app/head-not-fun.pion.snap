exit status: 0
stdout:
#error : #error

stderr:
error: tried to apply argument to non-function expression
  ┌─ tests/fail/fun-app/head-not-fun.pion:3:1
  │
3 │ Bool false
  │ ^^^^ ----- argument
  │ │     
  │ expression of type `Type`

